src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -2357,7 +2357,6 @@ internal void Run(bool inputLoopIsNested)
                 Dbg.Assert(ui != null, "Host.UI should return an instance.");
```

Comment:
@vexx32, your last commit had 2 failures in `PowerShell-CI-linux`
PowerShellGet - Module tests (Admin).Error occurred in Describe block
```powershell
At line:2 char:57
+             [System.Diagnostics.DebuggerHiddenAttribute()]
+                                                         ~
Unexpected token ')' in expression or statement.
At line:8 char:34
```
PowerShellGet - Script tests (Admin).Should install a script correctly to the required location with AllUsers scope
```powershell
At line:2 char:57
+             [System.Diagnostics.DebuggerHiddenAttribute()]
+                                                         ~
Unexpected token ')' in expression or statement.
at <ScriptBlock>, : line 8
8:             $foundSuggestion = @(Get-Command ($ExecutionContext.SessionState.Path.Combine(".", $escapedCommand)) -ErrorAction Ignore).Count -gt 0

```